### PR TITLE
Fixes typo with box of hugs message, adds one second cooldown to hugging it

### LINKED ISSUE #10692: The the box of hugs
1. The hug message says "You hug the the box of hugs."
2. It's very easy to spam the chat with the hug message
3. It comes with bruise packs, ointment and a medipen despite not being a medkit for some reason

### PR DESCRIPTION
**Pull request by freerealestate**
- Changes hug sound to the mob hug sound (seemed to fit the item better than the generic storage rustle)
- Tweaks a few of the descriptions in boxes.dm to read better, which I did while waiting for the code to compile

I don't mind changing any of the superfluous stuff back if people find a problem with it. The main thing is fixing the typo and adding the small cooldown to prevent spam.

Fixes #10692 

To do:
- [ ] Add cooldown to attack_self hotkey
- [ ] Remove object cooldown

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -624,13 +624,18 @@
...
-	..()
-	user.changeNext_move(CLICK_CD_MELEE)
-	playsound(loc, "rustle", 50, 1, -5)
-	user.visible_message("<span class='notice'>[user] hugs the [src].</span>","<span class='notice'>You hug the [src].</span>")
-	return
+	if (cooldown < world.time)
+		cooldown = (world.time + 10) // Sets cooldown at 1 second
+		..()
+		user.changeNext_move(CLICK_CD_MELEE)
+		playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -5)
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/storage/boxes.dm]: >thudswoosh
>not the hug sound

Comment (freerealestate) [inline code/game/objects/items/weapons/storage/boxes.dm]: It is, actually.

Comment (Iamgoofball) [inline code/game/objects/items/weapons/storage/boxes.dm]: welp

Comment (freerealestate) [inline code/game/objects/items/weapons/storage/boxes.dm]: I know, I had to check for myself.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -624,13 +624,18 @@
 	desc = "A special box for sensitive people."
 	icon_state = "hugbox"
 	foldable = null
+	var/cooldown = 0
 
 /obj/item/weapon/storage/box/hug/attack_self(mob/user)
-	..()
-	user.changeNext_move(CLICK_CD_MELEE)
```
Comment (Ikarrus) [inline code/game/objects/items/weapons/storage/boxes.dm]: Why not just use user.changeNext_move(CLICK_CD_MELEE+10)?

Comment (freerealestate) [inline code/game/objects/items/weapons/storage/boxes.dm]: Because I had no idea what that did, and just left it there because I
assumed it was needed.

I'll do that, thanks.

Comment (freerealestate) [inline code/game/objects/items/weapons/storage/boxes.dm]: That uh, that didn't change anything. You can still spam it.

Comment (Ikarrus) [inline code/game/objects/items/weapons/storage/boxes.dm]: Pretty sure CLICK_CD_MELEE is just a define for the actual click cooldown...

Oh unless it doesn't affect hotkeys.  That's probably it. 

Fucking hotkeys.

Comment (Ikarrus) [inline code/game/objects/items/weapons/storage/boxes.dm]: See if you can't fix this (and probably a lot of other stuff) by making the proc work with the use hotkey, too.

Comment (Ikarrus) [inline code/game/objects/items/weapons/storage/boxes.dm]: If I haven't made it clear, that proc is supposed to be a cooldown proc, to prevent spam/atatatata attacks.

Comment (freerealestate) [inline code/game/objects/items/weapons/storage/boxes.dm]: Okay, I'll see if I can figure it out.

Comment (RemieRichards) [inline code/game/objects/items/weapons/storage/boxes.dm]: @freerealestate Don't bother, the hotkey system we use is byond's inbuilt one, which points to a bunch of nonorganised procs that all fail to handle cooldowns.

I'm sure you'd probably be able to work it out for attack_self (this proc), but that's just a band aid fix.
When @Ikarrus suggested you fix cooldowns, I don't think they realised how messed up it currently is.

tl;dr it's going to be annoying as fuck, so feel free to not do it.

Comment (Ikarrus) [inline code/game/objects/items/weapons/storage/boxes.dm]: Yeah, if that's the case then fuck it.

**Comment by Aranclanos** [comment]
you're adding a object-based cooldown on a extremely simple item because you can spam it with hotkeys because they are bugged?

**Comment by freerealestate** [comment]
When I made this PR, I wasn't aware that hotkeys didn't have a cooldown the way that clicks do. And I figured you could spam it with clicks, too.

However, I'll probably do as Ikarrus suggested and see if I can't figure out a way to add click cooldown to the use in-hand object hotkey.

**Comment by Iamgoofball** [comment]
@freerealestate 

Don't. You'd have to cooldown the normal inhand click cooldown, and we delibrately removed cooldowns from all inventory management and item interaction that isn't combat related because it serves no purpose than to piss people off.

**Comment by freerealestate** [comment]
It'd slow down grab upgrading for hotkey players (it's very fast now that
you can use hotkeys for it), as well as making other in-hand use objects
harder to spam too. It's not the same as quick-equip, which would be
retarded if it had a cooldown.

However, I'm fine with sticking with the object cooldown. Less work for me.

**Comment by Aranclanos** [comment]
well to be fair it sound like stupidly hard task for someone starting like you, dunno about your experience before s133 though.
And also a lot of stupid simple shitty toys already have their own cooldown so whatvers

**Comment by Ikarrus** [comment]
Just guessing here, but it might be simpler to just make the cooldown affect attack_self procs instead of hotkeys, if you're determined to do this. 

Really, it's only the attack_self hotkey that would need a cooldown anyways. 

@iamgoofball we have similar cooldowns for spammable items already, like toys. In fact, it was aran who suggested I use his proc for this in the first place.
